src/common/components/report-issue-dropdown-menu.tsx

```diff
@@ -52,6 +54,9 @@ export const ReportIssueDropdownMenu = forwardRef< HTMLElement, Props >(
 				// In the future, if we re-use this dropdown apart from navigation from duplicate searching,
 				// we may have defer more of these actions to calling components.
 				dispatch( setIssueType( issueType ) );
+
+				monitoringClient.analytics.recordEvent( 'type_save', { issueType: issueType } );
```

Comment:
**Update:** Ahh, sorry, I probably wasn't super clear in the planning post... But I think we should reserve `type_save` for just capturing when the user saves the type by click the "continue" button in the Reporting flow. That name is now a little generic, but I want to avoid abandoning too many track events if possible (since we've already been recording this one for that purpose), even if the names aren't quite as descriptive as they could be.

Would you be okay just leaving this out until you add the `navbar_report_issue_start` event in  a future PR?